### PR TITLE
add `BW_DEBUG_LOG_DIR`

### PR DESCRIPTION
**Pull request by trehn**
#274

### CODE REVIEW

File: bundlewrap/utils/ui.py [hunk 1]
```diff
@@ -167,14 +193,19 @@ def deactivate(self):
...
     @clear_formatting
     @add_debug_indicator
+    @capture_for_debug_logfile
     @add_debug_timestamp
     def debug(self, msg, append_newline=True):
         if self.debug_mode:
             with self.lock:
                 self._write(msg, append_newline=append_newline)
+        if self.debug_log_file:
+            self.debug_log_file
```
Comment (vain) [inline bundlewrap/utils/ui.py]: What's this statement for?

Comment (trehn) [inline bundlewrap/utils/ui.py]: Whoopsie.

**Comment by vain** [comment]
Oh, yeah, output files are rather ... large. One run of `bw verify` on one node produced 440kb. I haven't tried our full repo yet.

Output format should probably be revised to make this more readable. Off the top of my head: The node name should always came right after `[DEBUG]`.

**Comment by trehn** [comment]
The worker pool stuff accounts for about one third of all lines, we can probably get rid of that.

**Comment by trehn** [comment]
As for output format, we should probably be more deliberate about *what* we're logging, too. But that's another issue IMO.

**Pull request merged**